testcases/single_cell_ispol/run_testcase.py

```diff
@@ -0,0 +1,35 @@
+import os
+from plot_testcase import plot_testcase
+from plot_testcase_forcing import plot_testcase_forcing
+
+#-------------------------------------------------------------------------------
+
+def run_testcase():
+
+    MPAS_SEAICE_EXECUTABLE = os.environ.get('MPAS_SEAICE_EXECUTABLE')
+    MPAS_SEAICE_TESTCASES_RUN_COMMAND = os.environ.get('MPAS_SEAICE_TESTCASES_RUN_COMMAND')
+    if (MPAS_SEAICE_TESTCASES_RUN_COMMAND is None):
+        MPAS_SEAICE_TESTCASES_RUN_COMMAND = ""
+    MPAS_SEAICE_DOMAINS_DIR = os.environ.get('MPAS_SEAICE_DOMAINS_DIR')
```

Comment:
Add check is set as here: https://github.com/MPAS-Dev/MPAS-Seaice_standalone_framework/blob/main/testcases/single_cell/run_testcase.py#L15